index.html

```diff
@@ -58,7 +60,221 @@ <h3>
 
 </section>
    
+
+<header class="title-header">
+    <h2 class="title">My Recent Works</h2>
+  </header>
+  <section class="recent-works">
+      <div class="work">
+          <section >
+  <img src="/img/Img-Placeholder.png" alt="">
+          </section>
+          <div class="mg">
+  
+        
+          <section class="stories">
+             Multi-Post Stories
+          </section>
+          <section class="story-details">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          </section>
+          <section class="langs">
+              <div>css</div>
+              <div>html</div>
+              <div>bootstrap</div>
+              <div>ruby</div>
+          </section>
```

Comment:
- [ ] This should be a list `<ul>` each language should be in a `<li>` kindly fix this.
- [ ] Kindly fix the same issue in all the other projects